build.gradle.kts

```diff
@@ -312,9 +312,9 @@ publishing {
     }
 }
 
-signing {
-    sign(publishing.publications["intershopMvn"])
-}
+//signing {
```

Comment:
Please remove comment marker so signing works again. To execute local builds without signing call gradle with `-x signI`.